xcode/Subconscious/Shared/Components/AddressBook/AddressBookEntryView.swift

```diff
@@ -9,13 +9,11 @@ import SwiftUI
 
 /// Full-sized user card, intended for listing users
 struct AddressBookEntryView: View {
-    var pfp: Image
     var petname: Petname
     var did: Did
     
     var body: some View {
         HStack(spacing: AppTheme.unit3) {
-            ProfilePic(image: pfp)
```

Comment:
Removed because we don't have this information here and don't need it, this view will go away in #478